help/de/Help_ch_Account-Actions.xml

```diff
@@ -902,20 +1037,24 @@
             </para>
           </listitem>
         </orderedlist>
-
-        <para>Wenn Sie die Kurse per Kommandozeile abrufen wollen, können Sie dazu den folgenden Befehl benutzen:
-        </para>
-
-        <para><code>gnucash --add-price-quotes &lt;gnucash-file-name&gt;</code>
+<!-- FIXME source: new footnote! -->
+        <para>Wenn Sie die Kurse per Kommandozeile abrufen wollen, können Sie dazu den Befehl
+          <command>gnucash-cli --quotes get <replaceable>gnucash-file-name</replaceable></command>
```

Comment:
`<replaceable><optional>pfad/zur/</optional>gnucash-datei</replaceable>`

unten auch. Falls öfter vorkommend in eine Entität packen.

Reply:
ok.